src/execution/index/art/node256.cpp

```diff
@@ -48,7 +57,8 @@ idx_t Node256::GetNextPos(idx_t pos) {
 }
 
 idx_t Node256::GetNextPosAndByte(idx_t pos, uint8_t &byte) {
-	for (pos == DConstants::INVALID_INDEX ? pos = 0 : pos++; pos < 256; pos++) {
+	pos == DConstants::INVALID_INDEX ? pos = 0 : pos++;
```

Comment:
nit:
I read this wrong before thinking you were assigning to `pos`, so the `pos++` looked weird to me
But then I saw that you don't assign to pos, they are just separate expressions

Can we use this instead as it's easier to understand for my smooth brain?
```suggestion
pos = (DConstants::INVALID_INDEX) ? 0 : pos + 1;
```